site/content/docs/5.3/migration.md

```diff
@@ -71,11 +72,17 @@ Learn more by reading the new [color modes documentation]({{< docsref "/customiz
   - The examples in our docs are now explicitly initialized and mostly don't use anymore `data-bs-ride`. Depending on how carousels are used in your project, it might need some updates.
   - <span class="badge text-danger-emphasis bg-danger">Breaking</span> The pause/play button is now included within the carousel for a better rendering but still respecting accessibility guidelines; it is now placed on the left-hand side of the indicators. The HTML markup has changed, please use this new version on your websites.
 
+- **Close button**
+  - Add tooltip on close button
```

Comment:
Maybe, to be precise: 
- A tooltip was added on close button in code examples. Please reflect this modification in your websites.
What do you think? (and yes it would be usefull on all modifications in this file)